.mention-bot

```diff
@@ -1,9 +0,0 @@
-{
-  "maxReviewers": 2,
```

Comment:
I've never seen a mention-bot comment in this repo. But weirly, I sometimes get notifications on Github, and I can see the mention-bot logo... Like it updated something somehow. :question: :question: